public/v4/apps/audiobookshelf.yml

```diff
@@ -0,0 +1,37 @@
+captainVersion: 4
+services:
+  $$cap_appname:
+    image: ghcr.io/advplyr/audiobookshelf:$$cap_version
+    ports:
+      - 13378:80
```

Comment:
Why do you need port mapping here?

Reply:
their docs perform this mapping, I thought they might have some internal config set to it.

https://www.[audiobookshelf](https://www.audiobookshelf.org/docs/#docker-install).org/docs/#docker-install

Reply:
This needs to be removed. The port mapping is only needed because the default config is not using reverse proxy.